classes/local/store/azure/client.php

```diff
@@ -27,18 +27,13 @@
 
 defined('MOODLE_INTERNAL') || die();
 
-use GuzzleHttp\Exception\ConnectException;
-use MicrosoftAzure\Storage\Blob\BlobRestProxy;
-use MicrosoftAzure\Storage\Common\Internal\Resources;
-use MicrosoftAzure\Storage\Common\ServicesBuilder;
-use MicrosoftAzure\Storage\Common\Exceptions\ServiceException;
 use SimpleXMLElement;
 use stdClass;
 use tool_objectfs\local\store\azure\stream_wrapper;
 use tool_objectfs\local\store\object_client_base;
 
 class client extends object_client_base {
-    const MAX_UPLOAD = Resources::MAX_BLOCK_BLOB_SIZE;
+    const MAX_UPLOAD = 5242880000000;
```

Comment:
Can you throw a comment here saying this is should be = to  Resources::MAX_BLOCK_BLOB_SIZE

or better yet can we make this just a variable instead of a const and set it in the constructor?

Reply:
Class variable has been created. I had to slightly refactor client and filesystem classes to meet this change

Reply:
Probably, this is not needed at all: https://github.com/catalyst/moodle-tool_objectfs/pull/285/commits/d3f95c491c0767c4ce6c3a0198e4a53eb694140a#diff-18ab1659be8a675b97cef923858847adR78-R83